### ISSUE TITLE
The error for "unset $KUBECONFIG" is needlessly cryptic

### ISSUE THREAD

**Issue by curusarn** [grafana/tanka#500]
Forgetting to set `$KUBECONFIG` results in a following error:
```
❯ tk diff envs/name
connecting to Kubernetes: finding usable context: expected clusters to be of type `[]map[string]interface{}`, but got `<nil>` instead
```

This is imho a pretty common mistake among newcomers so it would make sense to handle unset `$KUBECONFIG` as a separate error.

What do you think?

**Comment by captncraig** [comment]
I agree.. sort of? $KUBECONFIG not being set will default to the system default (`$HOME/.kube/config` or something), but if *that* is absent, yeah, probably makes sense to tell them we couldn't find anything.

**Comment by curusarn** [comment]
I'm willing to submit a PR fixing this if maintainers agree that it is a good idea.

**Comment by Duologic** [comment]
Sounds reasonable to me, a nice hint could help a newcomer definitely.
